app/views/content/life-as-a-teacher/explore-subjects/biology/_article.html.erb

```diff
@@ -1,44 +1,31 @@
 <div class="row inset">
   <section class="col col-720 col-space-l-top">
-    <h2 class="heading--box-blue">Why teach biology</h2>
+    <h2 class="heading--box-blue">Why teach <%= @front_matter["subject"] %></h2>
     <div class="inset">
-      <p>Biology is the study of living things, the intricate systems that sustain them, and the delicate balance of our ecosystems. As a biology teacher, you'll have the opportunity to inspire the next generation to explore the wonders of life itself.</p>
-      <p>In addition to the personal rewards of teaching biology, there are practical benefits too, including:</p>
-      <ul>
-        <li>a <a href="/life-as-a-teacher/pay-and-benefits/teacher-pay">competitive salary starting at <%= value :salaries_starting_minshortened %></a> (or higher in London)</li>
-        <li>a <a href="/life-as-a-teacher/pay-and-benefits/teachers-pension-scheme">generous and secure pension</a></li>
-        <li>more holiday days than people in many other professions</li>
-        <li>opportunities for progression</li>
-      </ul>
+      <p><%= @front_matter["subject"].titleize %> is the study of living things, the intricate systems that sustain them, and the delicate balance of our ecosystems. As a biology teacher, you'll have the opportunity to inspire the next generation to explore the wonders of life itself.</p>
```

Comment:
If we are replacing biology at the start of the sentence, shouldn't we replace it in the middle of the sentence?

(or alternatively, don't replace either as it makes it more complex 😄)

Reply:
On that note - would be worth you two thinking about whether replacing every instance of the subject on the page is worth it where the paragraphs are bespoke

There are quite a lot more references to `biology` on the page that **aren't** replaced with the variable

My view is that this it is not worth replacing in any bespoke paragraphs but I will leave it up to you too!